plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/TypeUtils.java

```diff
@@ -64,4 +91,93 @@ public static boolean isPushdownSupportedType(Type type)
                 || type instanceof ObjectIdType
                 || PUSHDOWN_SUPPORTED_PRIMITIVE_TYPES.contains(type);
     }
+
+    public static boolean isPushdownSupportedArrayElementType(Type type)
+    {
+        if (type == TIME_MILLIS || type == VARBINARY) {
+            return false;
+        }
+        return isPushdownSupportedType(type);
+    }
+
+    public static Optional<Object> translateValue(Object trinoNativeValue, Type type)
```

Comment:
Please separate a commit for extracting methods.

Reply:
done